src/traits/graph.rs

```diff
@@ -20,13 +20,23 @@ where
     }
 }
 
+impl<'a, G> ExactSizeIterator for SequentialGraphImplIter<'a, G>
+where
+    G: RandomAccessGraph
+        + SequentialGraph<SequentialSuccessorIter<'a> = G::RandomSuccessorIter<'a>>,
+{
+    fn len(&self) -> usize {
```

Comment:
Tommaso made me notice that this is wrong. The length should decrease as the iterator progresses.

Reply:
Ah right, I made the same mistake in one of my own crates too